<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetWakuTimer</class>
 <widget class="QDialog" name="GetWakuTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>タイマー設定</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>yy/M/d H:mm</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="getNow">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>now</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>タイマーを有効</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GetWakuTimer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GetWakuTimer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>getNow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
